includes/class-amp-service-worker.php

```diff
@@ -65,6 +65,27 @@ public static function init() {
 		}
 	}
 
+	/**
+	 * Register a caching route.
+	 *
+	 * @param WP_Service_Worker_Scripts $service_workers Service workers.
+	 * @param string                    $route           Route.
+	 * @param string                    $strategy        Strategy name.
+	 * @param array                     $args            Strategy args.
+	 * @param array                     $plugins         Plugins.
+	 */
+	private static function register_caching_route( WP_Service_Worker_Scripts $service_workers, $route, $strategy, $args = [], $plugins = [] ) {
+		$caching_routes = $service_workers->caching_routes();
+		if ( defined( 'PWA_VERSION' ) && version_compare( PWA_VERSION, '0.6', '<' ) ) {
+			$args['strategy'] = $strategy;
+			$args['plugins']  = $plugins;
+			$caching_routes->register( $route, $args );
```

Comment:
A really minor nitpick: This `register` method signature no longer matches the updated stub. I wonder if there is a way to indicate the previous method signatures on the stub.

Reply:
The new stub is:

```
register( $route, $strategy, $args = array() )
```

So doesn't it match? The `$args` has a default value of `array()` so it matches, right?

Reply:
![image](https://user-images.githubusercontent.com/134745/102550747-32a4a600-4073-11eb-9678-7a88bcd9864d.png)

![image](https://user-images.githubusercontent.com/134745/102550861-713a6080-4073-11eb-9a07-e665115b16e4.png)

The description for the `$strategy` actually indicates the deprecated usage, and it still allows `array`.

Reply:
I see that `$strategy` actually indicates the deprecated usage, so no worries here.